Eatery/Views/Eateries/HomescreenViews/EateryCollectionViewCell.swift

```diff
@@ -134,10 +134,10 @@ class EateryCollectionViewCell: UICollectionViewCell {
         infoContainer.snp.makeConstraints { make in
             make.leading.trailing.equalToSuperview()
         }
-        menuTextViewHiddenConstraints.append(contentsOf: infoContainer.snp.prepareConstraints { make in
+        favoriteHiddenConstraints.append(contentsOf: infoContainer.snp.prepareConstraints { make in
             make.bottom.equalToSuperview()
         })
```

Comment:
This spacing is off. It should be

```suggestion
        favoriteHiddenConstraints.append(
            contentsOf: infoContainer.snp.prepareConstraints { make in
                make.bottom.equalToSuperview()
            }
        )
```

Same thing below with all the other `append`s.